python/semantic_kernel/kernel.py

```diff
@@ -346,6 +346,72 @@ async def invoke_prompt(
         )
         return await self.invoke(function=function, arguments=arguments)
 
+    async def invoke_prompt_stream(
+        self,
+        function_name: str,
+        plugin_name: str,
+        prompt: str,
+        arguments: KernelArguments | None = None,
+        template_format: Literal[
+            "semantic-kernel",
+            "handlebars",
+            "jinja2",
+        ] = KERNEL_TEMPLATE_FORMAT_NAME,
+        return_function_results: bool | None = False,
+        **kwargs: Any,
+    ) -> AsyncIterable[list["StreamingContentMixin"] | FunctionResult | list[FunctionResult]]:
+        """
+        Invoke a function from the provided prompt and stream the results
+
+        Args:
+            function_name (str): The name of the function
+            plugin_name (str): The name of the plugin
+            prompt (str): The prompt to use
+            arguments (KernelArguments | None): The arguments to pass to the function(s), optional
+            template_format (str | None): The format of the prompt template
+            kwargs (dict[str, Any]): arguments that can be used instead of supplying KernelArguments
+
+        Returns:
+            AsyncIterable[StreamingContentMixin]: The content of the stream of the last function provided.
+        """
+        if not arguments:
+            arguments = KernelArguments(**kwargs)
+        if not prompt:
+            raise TemplateSyntaxError("The prompt is either null or empty.")
+
+        from semantic_kernel.functions.kernel_function_from_prompt import KernelFunctionFromPrompt
```

Comment:
Does this import needs to be here? Could it be at the module level?

Reply:
For typing, it does need to be here.